chatcommands.py

```diff
@@ -639,7 +668,7 @@ def tea(msg, other_user):
     """
 
     if other_user is None:
-        return "*brews a cup of {} tea for @{}*".format(random.choice(TEAS), msg.owner.name.replace(" ", ""))
+        return "*brews a cup of {} tea for {}*".format(random.choice(TEAS), msg.owner.name.replace(" ", ""))
```

Comment:
We had the `@` in here so that it'd actually *ping* the requester of the command.  This was done intentionally.